src/Components/getList.js

```diff
@@ -1,29 +1,57 @@
 import React from 'react';
 import { FirestoreCollection } from 'react-firestore';
 import { getLocalToken } from '../lib/token.js';
+import { useHistory } from 'react-router-dom';
 
-class GetList extends React.Component {
-  render() {
-    return (
-      <FirestoreCollection
-        path={`${getLocalToken()}/`}
-        render={({ isLoading, data }) => {
-          return isLoading ? (
-            <div>loading...</div>
-          ) : (
-            <div>
-              <h1>Groceries</h1>
-              <ul>
-                {data.map(item => (
-                  <li key={item.id}>{item.itemName}</li>
-                ))}
-              </ul>
-            </div>
-          );
-        }}
-      />
-    );
+function EmptyList() {
+  let history = useHistory();
```

Comment:
@nabilkaz @DeeThompson I'd like to know why did you choose `let` over `const` and `var` here 🙂